requirements.txt

```diff
@@ -7,4 +7,9 @@ pymongo==2.8
 Flask==0.10.1
```

Comment:
this file should be discarded.
We can use `setup.py` to track all third-party libs this package requires.